src/mono/netcore/sample/wasm/browser/runtime.js

```diff
@@ -9,7 +9,8 @@ var Module = {
         config.fetch_file_cb = function (asset) {
             return fetch (asset, { credentials: 'same-origin' });
         }
-
+        MONO.mono_wasm_init_aot_profiler();
```

Comment:
This shouldn't be needed.

Reply:
Ok, I just need to define `config.aot_profiler_options` to have initialize the profiler.

Reply:
Yes, the build task should set it in the config